dot/network/light.go

```diff
@@ -40,22 +40,17 @@ func (l *LightRequest) SubProtocol() string {
 }
 
 // Encode encodes a LightRequest message using SCALE and appends the type byte to the start
+// TODO Remove encode/decode receiver functions
```

Comment:
woudl this require updating the `Message` interface?

Reply:
Yes I believe so, I don't have to change it if I shouldn't. Was just thinking less receivers the better